test/xds/xds_client_custom_lb_test.go

```diff
@@ -91,12 +90,6 @@ func clusterWithLBConfiguration(clusterName, edsServiceName string, secLevel e2e
 // first) child load balancing policy, and asserts the correct distribution
 // based on the locality weights and the endpoint picking policy specified.
 func (s) TestWrrLocality(t *testing.T) {
-	oldCustomLBSupport := envconfig.XDSCustomLBPolicy
-	envconfig.XDSCustomLBPolicy = true
-	defer func() {
-		envconfig.XDSCustomLBPolicy = oldCustomLBSupport
-	}()
-
```

Comment:
Probably would be best to leave these in there until the env var is removed.  Otherwise the tests will fail if you disable it in your environment.

Reply:
Deleted these (by git checkout master ./) lol